src/components/pokedex.html

```diff
@@ -88,7 +88,7 @@ <h4 style="margin-bottom: 0">Pokédex</h4>
                        <div class="col-sm-2">
                         <label class="form-check-label">
                             <input class="form-check-input" type="checkbox" id="pokedex-filter-held-item"
-                                   value="false" checked onclick="PokedexHelper.toggleAllShiny(!PokedexHelper.toggleAllShiny())"> Hide shiny image
+                                   value="true" onclick="PokedexHelper.toggleAllShiny(!PokedexHelper.toggleAllShiny())"> Hide shiny image
```

Comment:
I've also just pushed this, as I forgot to remove the `checked` attribute when I flipped the setting name.

Reply:
I removed the checked and keep `false`, this `value` isnt used and for me represent if checkbox is checked or not